app/_layouts/extension.html

```diff
@@ -178,21 +239,37 @@
   {% if config_required_fields_yaml == 'config: ' %}config:
     <optional_parameter>: <value>{% else %}{{ config_required_fields_yaml }}{% endif %}
 ```
-{% endnavtab %}
-{% endunless %}
-{% endnavtabs %}
-{% endif %}
 
-`<service>` is the `id` or `name` of the Service that this plugin
+`<service>` is the `id` or `name` of the service that this plugin
   configuration will target.
 
-{% if page.params.api_id %}
-  > **Note:** The legacy API entity has been **deprecated** in favor of Services
-  since <a href="https://github.com/Kong/kong/blob/master/CHANGELOG.md#0130---20180322">
-  CE 0.13.0</a> and
-  <a href="https://docs.konghq.com/enterprise/changelog/#0-32">EE 0.32</a>.
-{% endif %}
+{% endnavtab %}
+{% endunless %}
+{% unless page.params.konnect_examples == false %}
+{% navtab Konnect %}
+Configure this plugin on a service:
+
+1. In Konnect, select the service on the ServiceHub page.
+2. Scroll down to **Versions** and select the version.
+3. Scroll down to **Plugins** and click **New Plugin**.
+4. Find and select the **{{ page.name }}** plugin.
+
+    <div class="alert alert-ee blue">
+    <strong>Note:</strong> If the plugin is greyed out, then it is not available
+    for your product tier. See <a href="">Kong Gateway tiers</a>.
+    </div>
```

Comment:
Not sure that you need this note in the Konnect tab at all. I've checked through all of my accounts - free, plus, trial - and the plugins that aren't available just don't appear. That might change in the future, but for now they're just missing.

Reply:
will remove